server/threadedserver.py

```diff
@@ -37,6 +47,8 @@ class ThreadedServer(SocketServer.ThreadingMixIn, SocketServer.TCPServer):
     allow_reuse_address = True
 
     def __init__(self, server_address, RequestHandlerClass):
+        #instantiate message handler to be used in the ThreadedRequestHandler
+        self.msg_handler = MessageHandler(1)
```

Comment:
this might be better off as a member of ThreadedRequestHandler but this is what I did for now, can easily change